src/main.rs

```diff
@@ -12,20 +15,19 @@ use log::{debug, error, info, trace, warn};
 use rand_chacha::ChaCha20Rng;
 use rand_core::{RngCore, SeedableRng};
 use std::cell::RefCell;
-use std::error::Error;
-use std::fmt;
 use std::str::FromStr;
 use tonic_lnd::{Client, ConnectError};
 
+include_config!();
```

Comment:
To silence clippy lints you can use:

```rust
use internal::*;

mod internal {
    #![allow(clippy::enum_variant_names)]
    #![allow(clippy::unnecessary_lazy_evaluations)]
    #![allow(clippy::useless_conversion)]

    include_config!();
}
```

Reply:
Thank you so much for the review.
this does not work, as The Config is not available in the main function.

I try to reference the commit https://github.com/Kixunil/configure_me/commit/7122f8e3350fcfdc30abb382f74462b93e76b7dd in Cargo.toml  using rev 7122f8e3350fcfdc30abb382f74462b93e76b7dd. 
I am getting the clippy warning still.

Reply:
What's the exact error message?

Reply:
```
error[E0433]: failed to resolve: could not find `config` in the list of imported crates
  --> src/main.rs:44:5
   |
44 |     include_config!();
   |     ^^^^^^^^^^^^^^^^^ could not find `config` in the list of imported crates
   |
   = note: this error originates in the macro `include_config` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0433]: failed to resolve: use of undeclared type `Config`
   --> src/main.rs:60:22
    |
60  |     let lnd_config = Config::including_optional_config_files(empty::<&str>())
    |                      ^^^^^^ use of undeclared type `Config`
```

Reply:
Oh, I think I know what's wrong. This should work until I fix it:

```rust
use internal::*;

mod internal {
    #![allow(clippy::enum_variant_names)]
    #![allow(clippy::unnecessary_lazy_evaluations)]
    #![allow(clippy::useless_conversion)]

    include!(concat!(env!("OUT_DIR"), "/configure_me_config.rs"));
}
```

Reply:
Works but I had to add it to the list.
` #![allow(clippy::never_loop)]`

Maybe consider ignoring it as well in the fix.
https://github.com/Kixunil/configure_me

:fireworks:

Reply:
Great! Yeah, I think I saw `never_loop` before.